modules/registrars/openprovider/OpenProvider/API/ApiV1.php

```diff
@@ -121,14 +123,67 @@ public function call(string $cmd, array $args = []): ResponseInterface
             return $response;
         }
 
+
+        if (isset($reply['warnings'])) {
+            $this->add_warning($reply['warnings'], $cmd, $args);
+        }
+
         $data = $this->serializer->normalize($reply->getData());
         $response = $this->successResponse($response, $data);
-
         $this->log($cmd, $args, $response);
 
         return $response;
     }
 
+    /**
+     * @param array $warnings
+     * @param string $cmd
+     */
+    private function add_warning($warnings, $cmd, $args)
```

Comment:
why this function name in snake case, check other functions eg `$this->successResponse`

Reply:
This change has been added